zeppelin-zengine/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java

```diff
@@ -449,6 +449,15 @@ public String getInterpreterLocalRepoPath() {
     return getRelativeDir(ConfVars.ZEPPELIN_INTERPRETER_LOCALREPO);
   }
 
+  public String getInterpreterRemoteRepoPath() {
+    String repo = getString(ConfVars.ZEPPELIN_INTERPRETER_LOCALREPO);
+    if (repo != null) {
```

Comment:
This conditional statement is unnecessary. getString() method returns default value if user didn't set environment variable.